### DISCUSSION TITLE
Use certificate from the Windows store ?

### DISCUSSION THREAD

**Discussion by vdebellabre** [mockoon/mockoon#1357] [Help]
Is there a way to make Mockoon use a certificate (for https) that resides in the Windows certificate store ?

Since I can't export them with their private key (security restriction from the company), that looks like my only option but I'm not sure this feature is supported.

**Comment by 255kb** [comment]
There is no support for the Windows cert store. 
We are relying on Node.js way of loading certificates (file based) and there is no plan to support another approach. 
Also, if you cannot export, I'm not sure if and how a Node.js library could access the certificates. Maybe it's not even easily doable.

**Reply by vdebellabre** [reply]
It is doable from Node since there are tools that do it, see https://www.npmjs.com/package/win-ca for example.
I'll try to find another way then, thanks for answering.

**Reply by 255kb** [reply]
yes, this is the package I checked, but my understanding was if you don't have the rights the npm libe shoudldn't be able to access the certificates either?

**Reply by vdebellabre** [reply]
My Windows account has the right to access the certificate and use it, I'm doing it with other processes.
But I don't have the rights to export that certificate from the store with it's private key, which is what is needed for Mockoon right now.
